Uplift/Controllers/HomeViewController.swift

```diff
@@ -53,7 +53,8 @@ class HomeViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        sections = [.myGyms, .todaysClasses, .lookingFor]
+        //MARK: removed .lookingFor
```

Comment:
Do you need to document why you removed this?

Reply:
yes